SEPACreditTransfer.pas

```diff
@@ -373,11 +377,18 @@ function TCreditTransferPaymentInformation.Validate(const schema: String; const
 procedure TCreditTransferPaymentInformation.SaveToStream(const stream: TStream; const schema: String);
 var
   i: Integer;
+    function _Boolean2Xml(_value:boolean):string;
+    begin
+        if _value then result:='true' else result:='false';
+    end;
```

Comment:
This function is defined twice in this PR. Please extract to the "SEPACommon.pas" file (and adapt indentation and whitespaces to the used code style, please also remove the "_" prefix in the variable name).